src/core/modules/SolarSystem.cpp

```diff
@@ -763,8 +763,16 @@ bool SolarSystem::loadPlanets(const QString& filePath)
 		// Note that for now we ignore any orbit-related config values from the ini file.
```

Comment:
// Note that for now we ignore any orbit-related config values from the ini file.

Reply:
Is this intended? Why? It makes more harm than good.